### PR TITLE
[SRVOCF-517] Add docs on setting custom PVC size for on-cluster function builds

### PR DESCRIPTION
**Pull request by rh-max**
Version(s):
`serverless-docs-1.30`+

Issue:
https://issues.redhat.com/browse/SRVOCF-517

Link to docs preview:
https://68625--docspreview.netlify.app/openshift-serverless/latest/functions/serverless-functions-on-cluster-builds#serverless-functions-setting-custom-volume-size_serverless-functions-on-cluster-builds

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

**Review by jrangelramos — COMMENTED** [review]
just one small note

**Comment by jrangelramos** [comment]
QE Approved

**Review by tmalove — APPROVED** [review]

**Comment by tmalove** [comment]
/remove-label peer-review-in-progress
/label peer-review-done

Don't forget to squash your commits.

**Comment by rh-max** [comment]
Thanks for reviews @jrangelramos , @tmalove !

File: modules/serverless-functions-setting-custom-volume-size.adoc [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+* You have installed the Knative (`kn`) CLI.
+
+.Procedure
+
+* Deploy your function with the `--pvc-size` flag and PVC size specification:
++
+[source,terminal]
+----
+kn func deploy --remote --pvc-size='2Gi'
```
Comment (jeana-redhat) [inline modules/serverless-functions-setting-custom-volume-size.adoc:25]:
```suggestion
$ kn func deploy --remote --pvc-size='2Gi'
```

File: modules/serverless-functions-setting-custom-volume-size.adoc [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+[id="serverless-functions-setting-custom-volume-size_{context}"]
+= Setting custom volume size
+
+For projects that require a volume with a larger size to build, you might need to customize the persistent volume claim (PVC) when building on the cluster. The default PVC size is 256 mebibytes.
+
+.Prerequisites
+
+* {pipelines-title} must be installed on your cluster.
+
+* You have installed the OpenShift (`oc`) CLI.
```
Comment (jeana-redhat) [inline modules/serverless-functions-setting-custom-volume-size.adoc:15]: There's an attribute for this now (assuming it works in serverless docs :thinking:)
```suggestion
* You have installed the {oc-first}.
```

File: modules/serverless-functions-setting-custom-volume-size.adoc [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+* You have installed the Knative (`kn`) CLI.
+
+.Procedure
+
+* Deploy your function with the `--pvc-size` flag and PVC size specification:
```
Comment (jeana-redhat) [inline modules/serverless-functions-setting-custom-volume-size.adoc:21]:
```suggestion
* Deploy your function with the `--pvc-size` flag and PVC size specification by running the following command:
```

**Review by jeana-redhat — COMMENTED** [review]
Just a couple tiny things

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: modules/serverless-functions-setting-custom-volume-size.adoc (hunk 2 above)
Comment (rh-max) [reply modules/serverless-functions-setting-custom-volume-size.adoc:15]: No, we don't have this attribute.

**Comment by jeana-redhat** [comment]
@rh-max thanks! LGTM

I don't think I have done a PR into serverless docs yet (or at least, not enough to remember).
1. Does this PR need any additional labels?
2. How does cherrypicking work? Specifically, if I need to CP after merge, what are the branch names to CP into?

**Comment by rh-max** [comment]
> @rh-max thanks! LGTM
> 
> I don't think I have done a PR into serverless docs yet (or at least, not enough to remember).
> 
> 1. Does this PR need any additional labels?
> 2. How does cherrypicking work? Specifically, if I need to CP after merge, what are the branch names to CP into?
> 
> Edit: oh yeah - also needs commits squashed first, please 🙇

@jeana-redhat Thanks!

1. I don't think any additional labels are necessary.
2. We do need CP after merge (the "Versions:" field above says "`serverless-docs-1.30`+"), and it is into the `serverless-docs-1.30` and `serverless-docs-1.31` branches.

**Comment by jeana-redhat** [comment]
@rh-max thanks! on it :grin:

**Pull request merged**